### DISCUSSION TITLE
Example for custom axes on canvas components

### DISCUSSION THREAD

**Discussion by narchibald** [plouc/nivo#2422] [Q&A]
Can anyone provide me with an example of how to implement custom axes for canvas components? In my case, it is for the Heat Map component, but I'm sure I can work out how to do it from and canvas graph component. What I'm trying to do is add extra information to the X-axis tick label eg. New Zealand    4.1M. Thanks

**Comment by Glidias** [comment]
instead of having to dive into the source code, they could at least have some documentation (or at least a link( on the website for this! ) 
https://github.com/plouc/nivo/blob/master/packages/axes/src/canvas.ts

but i guess it's assumed users would search on their own.
